scss/components/_soe_dm_article.scss

```diff
@@ -478,16 +480,6 @@
 .view-stanford-magazine-article-mag-landing-page,
 .view-stanford-magazine-collection-mag-landing-page,
 .view-display-id-article_collection_bottom_block {
-  .mag-article-container {
-    .mag-article-content-container {
-      .mag-article-title {
-        a {
-          font-weight: 600;
-        }
```

Comment:
Hi @minorwm,

I'm curious why you're deleting this font-weight?
I don't see any reference to it in the description for the PR

Reply:
Because it's not needed. It's already defined for the a tag
<img width="1258" alt="screen shot 2018-06-21 at 9 19 44 am" src="https://user-images.githubusercontent.com/571421/41725016-71a025e4-7534-11e8-8852-5acdb723121e.png">

Reply:
@minorwm,
Cool. Thanks for the verification.